org.flatpak.easydict-gtk.yml

```diff
@@ -0,0 +1,46 @@
+app-id: org.flatpak.easydict-gtk
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: easydict-gtk
+modules:
+  - name: poetry-core
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps .
+    sources:
+      - type: archive
+        url: https://github.com/python-poetry/poetry-core/archive/refs/tags/1.0.3.tar.gz
+        sha256: b5a9f8284c685c638796a322ea868843da5e2094c664436e44061e6bcea367ea
+  - name: orjson
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps orjson-3.6.4-cp38-cp38-manylinux_2_24_x86_64.whl
```

Comment:
this is an x86_64 specific wheel. python module should be built from source instead.

note: that you can use the flatpak-builder-tools to generate a manifest for the python modules.

Reply:
OK, I will have to limit this only to x86_64, because I don't want to build whole orjson module, because it has rust dependencies.

Reply:
aarch64 users will appreciate.

Reply:
on pypi is aarch64 orjson wheel, is there a possibility to use another wheel in manifest if the architecture is aarch64?

Reply:
only-arches: [aarch64] and only-arches: [x86_64] should be OK, nice :)